Azure.ResourceManager.Core/ResourceGroupContainer.cs

```diff
@@ -9,14 +9,24 @@
 namespace Azure.ResourceManager.Core
 {
     /// <summary>
-    /// Operations for the RespourceGroups container in the given subscription context.  Allows Creating and listign respource groups
+    /// Operations for the RespourceGroups container in the given subscription context.  Allows Creating and listing respource groups
```

Comment:
update summar to match https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#standard-class-documentation